spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -124,12 +124,54 @@ interface Packet {
 
 Note that a `Packet` is never directly serialised. Rather it is an intermediary structure used in certain function calls that may need to be created or processed by modules calling the IBC handler.
 
+Question: why is this here? Isn't there a protobuf definition for Packet serialization which must be understood by all
+chains that participate in IBC?
+
 An `OpaquePacket` is a packet, but cloaked in an obscuring data type by the host state machine, such that a module cannot act upon it other than to pass it to the IBC handler. The IBC handler can cast a `Packet` to an `OpaquePacket` and vice versa.
 
 ```typescript
 type OpaquePacket = object
 ```
 
+**Update:**
+
+The first version of IBC considered `Acknowledgement` simple application dependent bytes. This stood in contrast to the
+`Packet` types, which includes a number of well-defined metadata fields in addition to an application-dependent
+`data` field, which contains the opaque binary data. While discussing relayer fee incentivization the lack of including
+any protocol-level information in the acknowledgement came up as a limiting factor. Thus, we propose the following format
+for a "V2" acknowledgement. Information on how to transition between the two will be discussed below during the packet
+samples.
+
+```typescript
+interface Acknowledgement {
+  data: bytes
+  payOnSource: string  // Maybe<string> ???
+  forwardRelayer: string
+}
+```
+
+- The `data` is an opaque value which can be defined by the application logic of the associated modules. The contents are
+  identical to a v1 acknowledgement.
+- The `payOnSource` indicates an address on the **source chain** that the *forward relayer* can specify, which should receive
+  any existing reward for relaying the IbcReceivePacket.
+- The `forwardRelayer` field represents the signer of the IbcReceivePacket message. This is always set and equal to the
+  address on the **destination chain** for the *forward relayer*.
+
+Open discussion:
+
+- Is `forwardRelayer` useful info? This information is available without any work by the *forward relayer* and seems
+  more generic than just fee incentivization.
```

Comment:
> What is forwardRelayer meant for?

I threw it in as the info was there, and I thought it might be useful (future proofing).
Then again, it may be totally useless and premature optimization.

Happy not to include it as well.

Reply:
>  Can the relayer query/ decode the ack bytes and verify that the payOnSource is in fact its address (hasn't been altered)?

This info must be exposed in the events, so the relayer can provide this info on the remote side for proving.
I was a bit confused by commitments, so I haven't finalized the hashes and commitments below